### PR TITLE
Feature/login auth

### PR DESCRIPTION
**Pull request by AttFlederX**
Implement login page & auth services on frontend

### CODE REVIEW

**Review by OlesiaTvardovska — APPROVED** [review]

File: src/frontend/src/app/components/login/login.component.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+main {
+  max-width: 480px;
+  margin: auto;
+  background-color: #00256133;
```
Comment (Meownjik) [inline src/frontend/src/app/components/login/login.component.css:4]: Is it a correct color? HEX color usually has 3 or 6 digits, but here are 8.
UPD: It seems to be transparency, never faced such a notation

File: src/frontend/src/app/components/login/login.component.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    if (val.email && val.password && !this.isInProgress)
+    {
+      this.isInProgress = true;
+
+      this.userService.signIn(val.email, val.password)
+        .subscribe(res => {
+          this.isInProgress = false;
+
+          if (res !== undefined) {
+            this.router.navigateByUrl(``);
```
Comment (Meownjik) [inline src/frontend/src/app/components/login/login.component.ts:39]: What a strange apostrophe ( ` vs ' ), does it work OK?

**Review by Meownjik — APPROVED** [review]

File: src/frontend/src/app/components/login/login.component.css (hunk 1 above)
Comment (AttFlederX) [reply src/frontend/src/app/components/login/login.component.css:4]: Exactly, the '33' in the end is the alpha (transparency) value

File: src/frontend/src/app/components/login/login.component.ts (hunk 2 above)
Comment (AttFlederX) [reply src/frontend/src/app/components/login/login.component.ts:39]: `` quotes are used for string interpolation in typescript, so they are completely ok to use (even though for this empty string any quote type would be ok)

**Pull request merged**